src/FunctorOps.cpp

```diff
@@ -249,7 +249,7 @@ IntrinsicFunctors functorBuiltIn(std::string_view symbol, const std::vector<Type
     });
 }
 
-bool isValidFunctorOpArity(std::string_view symbol, size_t arity) {
+bool isValidFunctorOpArity(std::string_view symbol, std::size_t arity) {
```

Comment:
The files that use these should, strictly speaking, ensure to "#include <cstddef>"